app/src/main/java/com/taskmuse/app/ui/activity/MainActivity/MainActivity.java

```diff
@@ -57,4 +57,11 @@ public void alreadyUser(){
                 .addToBackStack(null) // Name can be null
                 .commit();
     }
+    public void redirectToDashboard(){
+        fragmentManager.beginTransaction()
+                .replace(R.id.fragmentContainerView, Login_Fragment.class, null)
```

Comment:
Here `Login_Fragment.class` should not be there I suppose as you are using this function to go to Dashboard. Correct? Or is it temporary?

Reply:
fixed - bad copy paste